harper-core/affixes.json

```diff
@@ -417,7 +417,9 @@
 				}
 			],
 			"target_metadata": {
-				"verb": {},
+				"verb": {
+					"tense": "Present"
```

Comment:
"Tense" is a syntactic property once words are in sentences and the sentences have been parsed. Words like "run", "walk", "eat" can be future if they come after "will", "shall", "going to", "gonna" or infinitive if they come after "to", etc.

The dictionary and affix system are lexical rather than syntactic. Terms used for this form of verbs are "dictionary form", "citation form", "lemma" - these three apply to other inflected parts of speech too. For verbs specifically "infinitive", literally "no time", "not inflected for tense".

Reply:
How did you make this comment here? Did GitHub mess up?